crafting_pieces.xml

```diff
@@ -5186,35 +5186,35 @@
   </CraftingPiece>
   <CraftingPiece id="crpg_polesword_blade_h0" name="{=kaikaikai}Exceptional Eastern Glaive" tier="4" piece_type="Blade" mesh="spear_blade_22" length="89" weight="0.44">
     <BladeData stack_amount="2" physics_material="wood_weapon" body_name="bo_spear_b">
-      <Thrust damage_type="Pierce" damage_factor="2" />
-      <Swing damage_type="Cut" damage_factor="5.2" />
+      <Thrust damage_type="Pierce" damage_factor="2.2" />
+      <Swing damage_type="Cut" damage_factor="4.3" />
     </BladeData>
     <Materials>

```

Comment:
Before the Polearm buff Polearm were considered universally trash. Here's where the polesword was.
![image](https://github.com/namidaka/itembalancing/assets/47090987/557a26a9-97fe-44bb-8382-e794c77baceb)
This nerf is insanely too strong. Polesword should be 4.9 IMO
Same for all others

Reply:
It's worth noting that the only reason this is the case is because Polesword is a low tier weapon:
"tier": 6.46829367,

The point of the change being that length should be a bigger factor in the stats a weapon receives. Polesword  would receive a change in the form of being made higher tier to bring its strength upward but also ensuring that its tier matches relative to it.

Longer weapons glance less, do more damage (because they glance less) and perform better overall compared to short polearms, It's also very capable of being used in a 1v1 despite its stats. Imho this change makes perfect sense.

Reply:
For context, the 'overall' nerf reduced polesword damage to 4.9. Further balancing by @Erdertus has nerfed polesword further than what was done by the changes to the polearm tier calculation.

Reply:
Firstly, I apologise, I should have branched from Droobs PR. This was a first stab at looking at how the stats would lie after the change. I was changing most weapons to roughly where they were Tier-wise by just changing damage values.

However, they look pretty inline to me. If anything I was thinking it could be even harsher, Swiss Halberd still looks scary, but play testing is better than looking at numbers on weapons.

Long polearms are objectively too strong ATM. Polesword has always been proof of that. The English Bill and Swiss Halberd are monsters.

I disagree with your statement that Polearms were considered universally trash pre-patch. Long swing polearms (Polesword, Burg Glaive, Long Bardiche, Voulge) were great pre-patch. 

Old polesword T6.7
New polesword T6.4

Polesword with 49c was a T9.X weapon.


A T10 Polesword would still have 49/50c with this change, at +0, which for a 246 reach weapon is still insanely good. So the polesword CAN have that much damage, just not at T6.X! 

Compare these values to 2h. They are still on the higher end of damage, and outreach a lot of 2h's.

Reply:
Most polearms have lost 3 or 4 damage. The longer polearms have lost more, to stay at the same tier, because the specific point was high damage, long polearms too value efficient.

Reply:
Most polearms have lost 2-4 damage to remain at the same tier. Longer polearms have lost more, shorter polearms lost less (might need to look at the bottom end, eg Norse Battle Axe, which could gain swing damage to stay at the same Tier, which I don't think is necessary).

Very long Polearms, especially those at lower Tiers, have lost more. Which was the point.

Reply:
> The point of the change being that length _should_ be a bigger factor in the stats a weapon receives. 

How do you know what change has been done to the itemvalue model. This is not in this Pull Request

> Longer weapons glance less, do more damage (because they glance less)

This is fundametely untrue.

Reply:
> For context, the 'overall' nerf reduced polesword damage to 4.9. Further balancing by @Erdertus has nerfed polesword further than what was done by the changes to the polearm tier calculation.

Then the nerf is perfectly in line with what i was envisionning. My issue is that your first commit is not done with itemvaluemodel change. So the tier decrease. 

>Firstly, I apologise, I should have branched from Droobs PR. This was a first stab at looking at how the stats would lie >after the change. I was changing most weapons to roughly where they were Tier-wise by just changing damage values.

This is the main source of the confusion.

Leave this Pull Request as is.

@Muparadzi 

Please do another pull request with just the damage nerf and the new itemmodel.
This pull request need to be done in conjunction with a Itemvalemodel pull request on the main repo.
Otherwise people will pay the upkeep according to the old value model.

Reply:
I've reverted @Erdertus so we just have the original itemmodel & damage nerf changes in this PR. The other PR has been raised on main repo.